specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/providers.json

```diff
@@ -158,6 +165,81 @@
           "description": "Blob duration of specification."
         }
       }
+    },
+    "MetricSpecification": {
+      "description": "Metric specification of operation.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of metric specification."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of Metric specification."
+        },
+        "displayDescription": {
+          "type": "string",
+          "description": "Display description of Metric specification."
+        },
+        "unit": {
+          "type": "string",
+          "description": "The metric unit. Possible values include: 'Bytes', 'Count', 'Milliseconds'."
+        },
+        "aggregationType": {
+          "type": "string",
+          "description": "The metric aggregation type. Possible values include: 'Average', 'Count', 'Total'."
+        },
+        "supportedAggregationTypes": {
+          "type": "array",
+          "description": "The supported aggregation types for the metrics.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "supportedTimeGrainTypes": {
+          "type": "array",
+          "description": "The supported time grain types for the metrics.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "lockAggregationType": {
+          "type": "string",
+          "description": "The metric lock aggregation type."
+        },
+        "dimensions": {
+          "type": "array",
+          "description": "The dimensions of metric",
+          "items": {
+            "$ref": "#/definitions/DimensionProperties"
+          }
+        },
+        "fillGapWithZero": {
+          "type": "boolean",
+          "description": "Property to specify whether to fill gap with zero."
+        },
+        "internalMetricName": {
+          "type": "string",
+          "description": "The internal metric name."
+        }
+      }
+    },
+    "DimensionProperties": {
+      "description": "Type of operation: get, read, delete, etc.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of dimension."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of dimension."
+        },
+        "toBeExportedForShoebox": {
+          "type": "boolean",
+          "description": "Property to specify whether the dimension should be exported for shoebox."
```

Comment:
Is "shoebox" a product name (first- or third-party)? If so, it should probably be capitalized depending on its marketing rules.